pom.xml

```diff
@@ -35,7 +35,7 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <ddf.version>2.13.4</ddf.version>
+        <ddf.version>2.13.5-SNAPSHOT</ddf.version>
```

Comment:
I dont think this should be committed. It should be pulled in automatically. I was having issues with this as well

Reply:
Generally, we just update this with the first person to merge a PR after a patch release, so should be fine.

Reply:
👍